doc/quickstart.rst

```diff
@@ -149,10 +146,10 @@ Good news!-- we can get the job done with just two lines of code. Here's the cod
         .cboreHole(2.4, 4.4, 2.1)
 
     # Render the solid
-    build_object(result)
+    show_object(result)
 
 
-After pressing F2 to re-execute the model, you should see something like this:
+After clicking the Redner button to re-execute the model, you should see something like this:
```

Comment:
Redner --> Render

Reply:
Thanks, I fixed that typo. I also turned on spell-checking in Sublime Text and found/fixed several others.